src/includes/sourcemaps/overview/javascript.remix.mdx

```diff
@@ -0,0 +1,15 @@
+Integrate your Remix project's source maps with Sentry using these steps:
+
+### 1: Generate Source Maps
+
+Remix can be configured to output source maps by Remix CLI. Learn more about how to generate sourcemaps with Remix [here](./generating/).
+
+### 2: Provide Source Maps to Sentry
+
+Source maps can be uploaded to Sentry by creating a release. Learn more about how to upload source maps [here](./uploading/).
+
+<Note>
+
+By default, if Sentry can't find the uploaded files it needs, it will attempt to download them from the URLs in the stacktrace. To disable this, turn off "Enable JavaScript source fetching" in either your organization's "Security & Privacy" settings or your project's general settings.
```

Comment:
```suggestion
By default, if Sentry can't find the uploaded files it needs, it will attempt to download them from the URLs in the stack trace. To disable this, turn off "Enable JavaScript source fetching" in either your organization's "Security & Privacy" settings or your project's general settings.
```